### PR TITLE
Update missing GroundDomain output variables and documentation

### PR DESCRIPTION
**Pull request by mitchute**
@mjwitte @Myoldmopar 

Addresses #5321

Please review output variable name changes. What sort of unit test would you recommend for this?

### CODE REVIEW

File: src/EnergyPlus/PlantPipingSystemsManager.cc [hunk 1]
```diff
@@ -2336,15 +2336,15 @@ namespace PlantPipingSystemsManager {
 
 		if ( PipingSystemDomains( DomainNum ).IsZoneCoupledSlab ) {
 			// Zone-coupled slab outputs
-			SetupOutputVariable( "Zone Coupled Surface Heat Flux [W/m2]", PipingSystemDomains( DomainNum ).HeatFlux, "Zone", "Average", PipingSystemDomains( DomainNum ).Name );
-			SetupOutputVariable( "Zone Coupled Surface Temperature [C]", PipingSystemDomains( DomainNum ).ZoneCoupledSurfaceTemp, "Zone", "Average", PipingSystemDomains( DomainNum ).Name );
+			SetupOutputVariable( "GroundDomain:Slab Zone Coupled Surface Heat Flux [W/m2]", PipingSystemDomains( DomainNum ).HeatFlux, "Zone", "Average", PipingSystemDomains( DomainNum ).Name );
```
Comment (mjwitte) [inline src/EnergyPlus/PlantPipingSystemsManager.cc]: @mmatts We generally avoid object class'ish names and colons in output variable names.  Better to use "Ground Domain Slab" or "GroundDomain Sab".

**Comment by mjwitte** [comment]
@mmatts Since I mixed this up by suggesting output variable name changes, this needs to wait until after v8.4.0 bug fix release, or be split into two separate pull requests.  (Sorry)  Either way, the name changes also need to be added to https://github.com/NREL/EnergyPlus/blob/develop/src/Transition/SupportFiles/Report%20Variables%208-4-0%20to%208-5-0.csv

**Comment by Myoldmopar** [comment]
@mmatts Here is the contents of the err file in ZoneCoupledGroundHTBasement:

```
Program Version,EnergyPlus, Version 8.4.0-0cdaffec21, YMD=2015.11.18 09:22,IDD_Version 8.4.0
   ** Severe  ** ReadBasementInputs:Site:GroundDomain:Basement="COUPLEDBASEMENT", invalid Basement Floor Boundary Condition Model Name="BASEMENTFLOOROSCM", Condition: Could not match with an Other Side Conditions Model input object.
   ** Severe  ** ReadBasementInputs:Site:GroundDomain:Basement="COUPLEDBASEMENT", invalid Basement Wall Boundary Condition Model Name="BASEMENTWALLOSCM", Condition: Could not match with an Other Side Conditions Model input object.
   ** Severe  ** Invalid Horizontal Insulation Material Name=BASEMENT INSULATION
   **   ~~~   ** Found in BASEMENT INSULATION
   ** Severe  ** Invalid Basement Wall Vertical Insulation Material Name=BASEMENT INSULATION
   **   ~~~   ** Found in BASEMENT INSULATION
   ** Severe  ** PipingSystems:ReadBasementInputs: Perimeter Horizontal Insulation Width is too high compared to basement floor dimensions
  <snip>
```

**Comment by mitchute** [comment]
:confused: 
I will look into it...

**Comment by mjwitte** [comment]
Probably an order of getinput issue which may be why you had this later in the first place.

**Comment by Myoldmopar** [comment]
@mmatts Am I getting my branches mixed up?  Did a ground-related branch have diffs in the csv files at one point?  This one appears fine, with the eio diffs only being that the lines are in different spots.  But I can't see the numeric diffs in the previous commit either.

**Comment by mitchute** [comment]
They were _tiny_ and only for the basement.

**Comment by Myoldmopar** [comment]
Aaaah, not crazy.  Well, maybe.  But in either case, this looks good now, no numeric diffs.

**Pull request merged**